TransformProcessor/qSlicerTransformProcessorModuleWidget.cxx

```diff
@@ -412,6 +419,8 @@ void qSlicerTransformProcessorModuleWidget::updateInputFieldVisibility()
     d->inputChangedTransformComboBox->setVisible( false );
     d->inputAnchorTransformLabel->setVisible( false );
     d->inputAnchorTransformComboBox->setVisible( false );
+    d->inputForwardTransformLabel->setVisible( false );
```

Comment:
A lot of code here. It would be much shorter if you called setVisible once for each widget. For example:
d->inputForwardTransformLabel->setVisible(average || shaftPivot)